docs/content.md

```diff
@@ -321,8 +321,25 @@ An example ERB implementation might be:
 </p>
 ```
 
+Values can also be utilised in the YAML-based internationalisation (I18n) translation definitions as used in the funding widget. The value should be specified as `%{value_name}`.
+
+An example YAML implementation might be: 
+
+```yaml
+en:
+  funding_widget:
+    subjects:
+      maths:
+        name: "Maths"
+        group: "Secondary"
+        sub_head: "Maths - starting salary: %{salaries_example_min}"
+        funding: "The opening date for applications is %{dates_example_opening} so get ready to apply soon."
+```
+
 A list of the current values available on the site can be viewed at the `/values` endpoint.
 
+Values should be named using only the characters `A` to `Z`, the numbers `0` to `9`, and the underscore `_` character. Unsupported characters such as the hyphen `-` are converted into underscores.
```

Comment:
@martyn-w, we spoke about formatting and rules earlier this week and thought that the best approach to take should be to use lowercase only. So I think we should make that clear here and not use uppercase letters in the example.

Reply:
good spot Myles, now updated